src/src/CustomBuild/define_plugin_sets.h

```diff
@@ -1356,7 +1356,7 @@ To create/register a plugin, you have to :
     #define USES_P073   // 7DGT
     #define USES_P079   // Wemos Motoshield
 
-    #if !defined(USES_P152) && defined(ESP32) && !(defined(ESP32C3) || defined(ESP32S3)) // Only supported on ESP32 and ESP32-S2
+    #if !defined(USES_P152) && defined(ESP32) && !(defined(ESP32C3) || defined(ESP32S3) || defined(ESP32C2) || defined(ESP32C6) || defined(ESP32H2)) // Only supported on ESP32 and ESP32-S2
```

Comment:
You can check for `ESP32_CLASSIC`, exactly for these kind of checks :)

Reply:
But that will only available after the C3/S3 PR is merged, I assume?

Reply:
Yep... that's true...